ProcessMaker/Models/Script.php

```diff
@@ -79,10 +80,14 @@ class Script extends ProcessMakerModel implements ScriptInterface
 
     protected $casts = [
         'timeout' => 'integer',
+        'retryAttempts' => 'integer',
+        'retryWaitTime' => 'integer',
```

Comment:
These should be retry_attempts and retry_wait_time to match the database columns

Reply:
fixed